app-emulation/docker/files/dockerd

```diff
@@ -34,8 +34,8 @@ parse_docker_args "$@"
 USE_SELINUX=""
 # Do not override selinux if it is already explicitly configured.
 if [[ -z "${ARG_SELINUX}" ]]; then
-        # If unspecified, default off
-        USE_SELINUX="--selinux-enabled=false"
+        # If unspecified, default on
+        USE_SELINUX="--selinux-enabled"
```

Comment:
I guess you need to take the latest change: https://github.com/kinvolk/coreos-overlay/commit/1f33d2f9745da2e38ec65079a6448eda6507ed26

Reply:
Yes, I was aware of that change but it got lost in the rebase.